src/index.ts

```diff
@@ -81,10 +81,11 @@ export async function generateFromMessages<T extends ExportReturnType = ExportRe
  * @param options The options to use when creating the transcript
  * @returns       The generated transcript
  */
+
 export async function createTranscript<T extends ExportReturnType = ExportReturnType.Attachment>(
   channel: TextBasedChannel,
   options: CreateTranscriptOptions<T> = {}
-): Promise<ObjectType<T>> {
+): Promise<ObjectType<T> | ObjectType<T>[]> {
```

Comment:
This isn't great since now everything returns `something | something[]` which means all typescript users will need add a `as <what they expected>` statement.